
#Parameters for the espeleo_planning package

# Curve number
N_curve: 4
# 1 -> Ellipse
# 2 -> Eigth like curve
# 3 -> Square like curve
# 4 -> Sine curve

# Number of points to be sampled
N_points: 200

# Stretch factor of the curve in the x direction
a: 2.8
# Stretch factor of the curve in the y direction
b: 2.8
# Rotation of the curve around the z axis (in dregrees)
phi: 0.0
# Displacement in the x direction (meters)
cx: 1.0
# Displacement in the y direction (meters)
cy: 1.5


#Flag to indicate if the path is closed or not
closed_path_flag: False
#Number of points to be inserted in between each pais of points of the received trajectory
insert_n_points: 10
#Number of points to use in the average filter (it is forced to be an odd number) - if 0 the path is not filtered
filter_path_n_average: 10



#Chose Dijkstra trajectory
dijkstra_traj_number: 1
# 1 -> Dijkstra Points Combined
# 2 -> Dijkstra Points Energy
# 3 -> Dijkstra Points Shortest
# 4 -> Dijkstra Points Transversal
